sparkles/tests/test_checks.py

```diff
@@ -56,6 +86,8 @@ def test_guide_count_er():
     assert msg['category'] == 'critical'
     assert 'ER count of 9th' in msg['text']
 
+
+def test_guide_count_er2():
```

Comment:
Note that as long as the setup is not being repeated it is nicer to split tests into atomic units.